### PR TITLE
Create Meet Erik Neemann post

### PR DESCRIPTION
**Pull request by atribolet**
Pull request description
------------------------

introduce erik neemann

### CODE REVIEW

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+author:
+  display_name: Erik Neemann
+  email: eneemann@utah.gov
+date: 2018-12-10 17:57:17
+categories:
+  - Featured
+tags: []
+---
+
+#What’s your title at AGRC?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**What’s your title at AGRC?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Technical Support Specialist II
+
+#(So what exactly do you do?)
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**(So what exactly do you do?)**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Initially, I’ll be attempting to fill the void left by the venerable David Buell as he retires after over 26 years of public service! This will entail a lot of work with 911 data to support many of the State’s Public Safety Answering Points (PSAPs). I will help manage law enforcement, fire, and emergency service boundaries, as well as road system and addressing data. These data sets require a lot of maintenance and massaging to ensure that they are accurate and up-to-date and that they seamlessly integrate with PSAP software to enable efficient call-taking and dispatching. Beyond the 911-centric work, I’ll be helping out with The Utah Reference Network (TURN) GPS and any other GIS projects that need support.
+
+#What’s your background?/How did you end up at AGRC?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**What’s your background?/How did you end up at AGRC?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+My road to the AGRC is a winding one. I originally started out in the meteorology field and spent ten years on active duty as an Air Force weather officer. After assignments in Arizona and Missouri, the Air Force sent me to the University of Utah for a couple of years to earn an MS in atmospheric science. When I finished that degree, I moved back to my native Nebraska to work on improving the science behind the Air Force’s weather forecast models at Offutt Air Force Base. During that time, I started to see the power of GIS and earned a GIS certificate online. After getting married, I decided to settle down and left the Air Force for civilian life back in the beautiful state of Utah! I took that opportunity to make a career change, and now I’m working on an MS in geographic information science. I eventually landed at the AGRC after a few different internships in the GIS field over the last two years (including one here). I’m excited to bring my scientific and professional background into a relatively new career and continually expand my knowledge among the GIS stalwarts at the AGRC.
+
+#GIS is a unique field; what made you decide to pursue a career in this field?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**GIS is a unique field; what made you decide to pursue a career in this field?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+#GIS is a unique field; what made you decide to pursue a career in this field? 
+
+As an undergraduate student and budding meteorologist in the 2000s, I became accustomed to looking at disparate weather data sources (e.g., radar, satellite, surface observations, and models) on different screens and in different interfaces. However, that old paradigm started to change in the last decade when GIS-based solutions brought everything into the same map. This shift made me realize how useful GIS solutions could be for data analysis and visualization, and my interest grew from there. I saw that similar developments were occurring with Air Force flight planning tools and I eventually decided to pursue a new career in the exciting GIS field after my time in the military was complete.
+
+#What’s the most interesting GIS project you’ve ever worked on, and why?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**What’s the most interesting GIS project you’ve ever worked on, and why?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+#What’s the most interesting GIS project you’ve ever worked on, and why? 
+
+I’ve had several interesting GIS projects that I’ve really enjoyed while taking courses over the last few years. These range from mapping out trips as an Uber driver to estimating debris-flow probability following the Thomas Fire in southern California. However, my personal favorite was identifying the best place to watch the August 21, 2017, total solar eclipse. For the project, I mapped out a weighted eclipse suitability score that accounted for the eclipse’s path of totality, duration of totality, average August cloud cover, vegetation/land cover type, and population density. I also updated the map with recalculated scores based on cloud forecast data from the National Weather Service a few days before the event. According to my original climate-based results, the panhandle of Nebraska was the best viewing location, but my actual experience suggests that Howe, Idaho, was a great viewing location too!
+
+#What do you want to accomplish at AGRC?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**What do you want to accomplish at AGRC?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+#What do you want to accomplish at AGRC? 
+
+I hope to be able to contribute to a wide variety of projects in which the AGRC is involved and learn from the vast experience and skills of my coworkers along the way. Specifically, I want to improve the labor-intensive 911 data preparation process with automation. This would ultimately lead to greater work efficiency, as well as increased data accuracy and timeliness. I also want to help guide the State’s efforts to adopt the Next Generation 911 (NG911) data standards and best practices, so we can further enhance emergency services and operate within the new standard. Beyond that, I just want to contribute in any way I can to leverage the power of GIS and improve processes wherever possible.
+
+#How do you spend your time outside the office? Any favorite hobbies or activities?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**How do you spend your time outside the office? Any favorite hobbies or activities?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+#How do you spend your time outside the office? Any favorite hobbies or activities? 
+
+My wife and I have a three-month-old daughter, so we now spend most of our time adoring her. We also like to get outside as much as possible to explore the state’s amazing scenery, trails, parks, and campgrounds, like most Utahns. Depending on the season, I like to snowboard and explore slot canyons while always keeping a watchful eye on the weather. Otherwise, I can usually be found running, golfing, or watching sports, especially Nebraska football and the NBA.
+
+#Favorite Utah trail/hike?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**Favorite Utah trail/hike?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+My wife and I have a three-month-old daughter, so we now spend most of our time adoring her. We also like to get outside as much as possible to explore the state’s amazing scenery, trails, parks, and campgrounds, like most Utahns. Depending on the season, I like to snowboard and explore slot canyons while always keeping a watchful eye on the weather. Otherwise, I can usually be found running, golfing, or watching sports, especially Nebraska football and the NBA.
+
+#Favorite Utah trail/hike? 
+
+Almost anything involving a peak or a canyon is a hike that I’ll enjoy. One of my favorites has to be the iconic Angel’s Landing hike in Zion National Park. It’s just about the right distance and has a great combination of scenery, danger, and difficulty. It can get crowded though, so if you want to explore that hike, I’d recommend taking the first bus into the canyon to beat the masses and decrease the number of times you have to squeeze by someone on the edge of a cliff. 
+The state has such a great variety of trails; there’s something for everyone, no matter what level of difficulty or remoteness you’re seeking.
+
+#Favorite food?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**Favorite food?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+#Favorite Utah trail/hike? 
+
+Almost anything involving a peak or a canyon is a hike that I’ll enjoy. One of my favorites has to be the iconic Angel’s Landing hike in Zion National Park. It’s just about the right distance and has a great combination of scenery, danger, and difficulty. It can get crowded though, so if you want to explore that hike, I’d recommend taking the first bus into the canyon to beat the masses and decrease the number of times you have to squeeze by someone on the edge of a cliff. 
+The state has such a great variety of trails; there’s something for everyone, no matter what level of difficulty or remoteness you’re seeking.
+
+#Favorite food? 
+
+I love most food, although my wife would point out some notable exceptions. Pizza and burgers are probably my favorite, but pretty much all of the unhealthy stuff tastes great to me!
+
+#Random skill that people don’t know you have?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**Random skill that people don’t know you have?**
```

File: _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+I can juggle a little bit.
+
+#Anything else?
```
Comment (steveoh) [inline _posts/2018-12-10-meet-erik-neemann-new-agrc-staff.md]:
```suggestion
**Anything else?**
```

**Comment by steveoh** [comment]
I think a nice blurb before the questions and this is ready to go.

**Comment by stdavis** [comment]
Line 32 still needs to be fixed...

**Comment by atribolet** [comment]
K added the intro and (I think) I fixed the formatting issue!

**Pull request merged**